.github/dependabot.yml

```diff
@@ -1,5 +1,4 @@
 ---
-
 # Any ignore directives should be uncommented in downstream projects to disable
```

Comment:
Do you have an IDE setting betraying you on this?

```suggestion

# Any ignore directives should be uncommented in downstream projects to disable
```

Reply:
I have no idea - I will keep an eye on this in the future.  Thanks for noting all of them!